adds a trash heap exoplanet ruin

Tazaraki opened the pull request:
🆑 Taza, gy1ta23
rscadd: new exoplanet ruin: trash heap
/🆑

I'm taking a stab at mapping stuff for the overmap and exoplanets. This was me following the wiki guide and seeing if I could get something simple but interesting to stumble upon added into the game before tackling larger projects.  

This 10x10 map is mostly randomly generated trash with a few select randomly generated items that explorers and exoplanet dwellers can stumble upon.

SierraKomodo commented:
Self-reminder to double-check this fits with the generalized/randomized planet map changes (And update the wiki accordingly if needed).

SierraKomodo reviewed (dismissed):
This needs to be run through mapmerge: https://github.com/Baystation12/Baystation12/wiki/Setting-up-mapmerge-2

Tazaraki committed:
adds a trash heap exoplanet ruin

lots of trash and a few goodies
adds new line at end of file


undoes an edit i didnt mean to do


forgot to commit me ticking this in the .dme


Maybe did the map merge thing


removes a snack from the trash pile

Tazaraki commented:
I believe I've successfully used mapmerge. 

As for the wiki, I can confirm that the section "Assignment, Testing and Travis" no longer applies as about a week after that wiki article was last edited someone made a change to have exoplanet ruins use a tag system instead of the list method described.
